### PR TITLE
Nbo/no scenario yet

### PR DESCRIPTION
**Pull request by Nikeleos**

### CODE REVIEW

File: src/views/Scenario/Scenario.js [hunk 1]
```diff
@@ -121,14 +127,13 @@ const Scenario = (props) => {
...
         <Card style={{ height: '400px' }}>
-          { currentScenario.data &&
             <Dashboard
               iframeTitle={t('commoncomponents.iframe.scenario.results.card.title', 'Results')}
-              url={SCENARIO_DASHBOARD_CONFIG.url.replaceAll('<ScenarioName>', currentScenario.data.name)}
-              scenarioName={currentScenario.data.name}
-              scenarioState={currentScenario.data.state}
+              url={SCENARIO_DASHBOARD_CONFIG.url.replaceAll('<ScenarioName>', currentScenario.data?.name)}
+              scenarioName={currentScenario.data?.name}
+              scenarioState={currentScenario.data?.state}
```
Comment (COSMO-VMI) [inline src/views/Scenario/Scenario.js:134]: What if currentScenario is undefined? is the `?` at the right place?

File: src/views/Dashboards/Dashboards.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ function a11yProps (index) {
 const Dashboards = ({ currentScenario }) => {
   const classes = useStyles();
   const [value, setValue] = useState(0);
+  const noScenario = currentScenario === null;
```
Comment (COSMO-VMI) [inline src/views/Dashboards/Dashboards.js:38]: `noScenario` seems to be declared as a bool (not sure, without parenthesis tho)...

File: src/views/Dashboards/Dashboards.js [hunk 3]
```diff
@@ -117,7 +119,8 @@ TabPanel.propTypes = {
   src: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
   scenarioName: PropTypes.string,
-  scenarioId: PropTypes.string
+  scenarioId: PropTypes.string,
+  noScenario: PropTypes.array.isRequired
```
Comment (COSMO-VMI) [inline src/views/Dashboards/Dashboards.js:123]: ... whereas it seems to be defined as a required array here.

File: public/locales/fr/translation.json [hunk 4]
```diff
@@ -94,14 +94,18 @@
...
             "no": {
               "result": "Pas de résultats pour ce scénario"
             },
             "error": "Une erreur est survenue lors de l'exécution de ce scénario",
             "running": "Scénario en cours d'exécution...",
             "uninitialized": "Le scénario n'a pas encore été lancé"
           }
+        },
+        "noscenario": {
+          "title": "Aucun scénario n'a encore été crée",
```
Comment (COSMO-VMI) [inline public/locales/fr/translation.json:107]: "créé"

**Review by COSMO-VMI — CHANGES_REQUESTED** [review]
Some fixes to make, the rest seems good to me.

File: src/views/Dashboards/Dashboards.js (hunk 3 above)
Comment (jreynard-code) [reply src/views/Dashboards/Dashboards.js:123]: Could you please fix the console error "Warning: Failed prop type: Invalid prop `noScenario` of type `boolean` supplied to `TabPanel`, expected `array`." ?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
Some changes but good work ;)

File: src/views/Scenario/Scenario.js (hunk 1 above)
Comment (Nikeleos) [reply src/views/Scenario/Scenario.js:141]: cuurentSecnrio is set from a reducer and cannot be null

File: src/views/Dashboards/Dashboards.js (hunk 2 above)
Comment (Nikeleos) [reply src/views/Dashboards/Dashboards.js:38]: Done

File: src/views/Dashboards/Dashboards.js (hunk 3 above)
Comment (Nikeleos) [reply src/views/Dashboards/Dashboards.js]: Done

File: public/locales/fr/translation.json (hunk 4 above)
Comment (Nikeleos) [reply public/locales/fr/translation.json]: Fixed

**Pull request merged**